activemodel/lib/active_model/dirty.rb

```diff
@@ -178,7 +178,7 @@ def changed_attributes
 
     # Handles <tt>*_changed?</tt> for +method_missing+.
     def attribute_changed?(attr, from: OPTION_NOT_GIVEN, to: OPTION_NOT_GIVEN) # :nodoc:
-      changes_include?(attr) &&
+      !!changes_include?(attr) &&
```

Comment:
I can see that the test is [definitely fails](https://travis-ci.org/rails/rails/jobs/122863305#L480),
but why? How can it return nil?